dsgrid/config/project_config.py

```diff
@@ -139,6 +143,7 @@ class InputDatasetModel(DSGBaseModel):
         default=DatasetRegistryStatus.UNREGISTERED,
         dsg_internal=True,
         notes=("status is "),
+        updateable=False,
```

Comment:
Does this mean that status is not directly updatable by users?

Reply:
Correct. Updated by dsgrid.

Reply:
I think that is the behavior we want. If it ever comes down to it and we need users to be able to update a status, then we will need to make a CLI command for that. But for now this is good.